From effac6d0ec264c7e9d859fe157f6aa2e71598f3d Mon Sep 17 00:00:00 2001
From: Robby Cai <R63905@freescale.com>
Date: Thu, 25 Nov 2010 15:54:23 +0800
Subject: [PATCH 195/505] ENGR00134055-2 Update test instances number to 15

In theory, it can virtually support up to 16 instances.
For safety, set to 15 because the EPDC driver occupies one.

Signed-off-by: Robby Cai <R63905@freescale.com>
---
 test/pxp_lib_test/pxp_test.c | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/test/pxp_lib_test/pxp_test.c b/test/pxp_lib_test/pxp_test.c
index 8638767..09671e5 100644
--- a/test/pxp_lib_test/pxp_test.c
+++ b/test/pxp_lib_test/pxp_test.c
@@ -71,6 +71,8 @@ char *usage = "Usage: ./pxp_test.out "\
 #define WIDTH	480
 #define HEIGHT	360
 
+#define NR_INSTANCE		15
+
 int fd_fb = 0;
 unsigned short * fb0;
 int g_fb0_size;
@@ -88,7 +90,7 @@ struct input_argument {
 sigset_t sigset;
 int quitflag;
 
-static struct input_argument input_arg[4];
+static struct input_argument input_arg[NR_INSTANCE];
 static int instance;
 
 static char *mainopts = "HI:";
@@ -115,7 +117,7 @@ int parse_main_args(int argc, char *argv[])
 			status = -1;
 			break;
 		}
-	} while ((opt != -1) && (status == 0) && (instance < 4));
+	} while ((opt != -1) && (status == 0) && (instance < NR_INSTANCE));
 
 	optind = 1;
 	return status;
-- 
1.8.0

